Automattic/wp-calypso
# Playwright: add unit tests for calypso-e2e module

#### Changes proposed in this Pull Request

This PR partially implements #53033.


#### Testing instructions

1. check out the branch.

either:
2. navigate to `packages/calypso-e2e`
3. run `yarn jest`

or
2. run `yarn test-packages`

Note, that if you have `NODE_CONFIG_ENV` environment variable set, tests will trigger a warning from `config` that it could not find any configuration. This is expected and should not matter since I ended up mocking the call anyway. If this should be suppressed, I am sure there is a way to do so with a mock, but in the TeamCity environment the aforementioned environment variable isn't set anyway so the warning doesn't even show up.

Related to #53033.
